private/bufpkg/bufimage/bufimagemodify/bufimagemodify.go

```diff
@@ -219,3 +247,27 @@ func int32SliceIsEqual(x []int32, y []int32) bool {
 	}
 	return true
 }
+
+func stringOverridesToBoolOverrides(stringOverrides map[string]string) (map[string]bool, error) {
+	validatedOverrides := map[string]bool{}
```

Comment:
We know the size of the map - let's use it.
```suggestion
	validatedOverrides := make(map[string]bool{}, len(stringOverrides))
```

This will apply to the other validation helpers when/if they exist (i.e. the `OptimizeFor` logic above).